tests/device_tests/test_fuel_gauge.py

```
@@ -0,0 +1,69 @@
+"""UI tests for the fuel gauge (battery indicator) on the Eckhart homescreen.
+
+Each test sets an emulated battery state and captures the homescreen appearance
+via the screen-recording infrastructure (--ui=record / --ui=test).
+"""
+
+import pytest
+
+from trezorlib import messages
+from trezorlib.debuglink import DebugSession as Session
+
+# T3W1 (TS7) only – the only model with a fuel gauge on homescreen
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)